src/Dev/Tasks/CleanupTestDatabasesTask.php

```diff
@@ -23,15 +23,11 @@ class CleanupTestDatabasesTask extends BuildTask
 
     public function run($request)
     {
-        if (!Permission::check('ADMIN') && !Director::is_cli()) {
```

Comment:
We should probably keep this (just replace it with `if (!$this->canView())` so we're still checking permissions for any weird scenario where this gets called directly, and giving the same result we were giving before.